rmgpy/statmech/__init__.py

```diff
@@ -28,8 +28,12 @@
 #                                                                             #
 ###############################################################################
 
-from .translation import Translation, IdealGasTranslation
-from .rotation import Rotation, LinearRotor, NonlinearRotor, KRotor, SphericalTopRotor
-from .vibration import Vibration, HarmonicOscillator
-from .torsion import Torsion, HinderedRotor
-from .conformer import Conformer
+"""
+initialize imports
+"""
+
+from rmgpy.statmech.translation import Translation, IdealGasTranslation
+from rmgpy.statmech.rotation import Rotation, LinearRotor, NonlinearRotor, KRotor, SphericalTopRotor
+from rmgpy.statmech.vibration import Vibration, HarmonicOscillator
+from rmgpy.statmech.torsion import Torsion, HinderedRotor
+from rmgpy.statmech.conformer import Conformer
```

Comment:
Could you reorder the imports alphabetically? i.e., start with the `.conformer` one, etc.
Also, take a look at other files in this PR?

Reply:
I changed imports accordingly. I split the imports into 3 sections: (1) future (2) numpy, os, unittest,etc. (3)rmg related imports. I change the sequence of numpy and unittest. If you think it is inappropriate, I can change it back.